components/identity-mgt/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/store/RegistryRecoveryDataStore.java

```diff
@@ -261,6 +271,5 @@ private void deleteOldResourcesIfFound(Registry registry, String userName, Strin
         } catch (RegistryException e) {
             log.error("Error while deleting the old confirmation code \n" + e);
```

Comment:
Same as above, better to trace the exception without concatenating it to the message.